frontend/src/App.js

```diff
@@ -1,6 +1,9 @@
 import logo from './logo.svg';
 import StartGameButton from './components/StartGameButton'
 import './App.css';
+import CountdownTimer from "./components/CountdownTimer";
```

Comment:
I believe this import is necessary to implement the countdown timer. Without it, the build will not run due to an error.

Reply:
The import is greyed out by the IDE because it's not used in this file.

<img width="697" alt="Screenshot 2023-10-01 at 11 33 41 AM" src="https://github.com/codeday-hom/hit-or-miss/assets/1451027/f905446d-fc26-4fad-9e6b-910725d687eb">